libethcore/Transaction.cpp

```diff
@@ -112,9 +112,9 @@ Address const& TransactionBase::sender() const
 	if (!m_sender)
 	{
 		auto p = recover(m_vrs, sha3(WithoutSignature));
```

Comment:
If we have a zero-signature, this calls `recover` for each call of `sender`. Please cache in both situations.